COMMITTING.md

```diff
@@ -0,0 +1,55 @@
+# WHATWG committer guidelines
+
+## Branches
+
+Branches are transient and their names must be detailed enough to avoid confusion. Choose short and descriptive names. Using hyphens to separate words is encouraged _(but not required)_.
+
+```bash
+# good
+$ git checkout -b descriptive-name-relative-to-feature
+```
+
+```bash
+# bad - too vague and malformatted
+$ git checkout -b fix_paragraph-header
+```
+
+Identifiers from corresponding Github issues are great candidates for use in branch names. For example, when working on an issue:
+
+```bash
+$ git checkout -b issue-15
+```
+
+## Committing
+
+Each change needs to result in a single commit on the master branch, with no merge commits. The green squash and merge button is OK to use, but be sure to tidy up the commit message per [guidelines for writing good commit messages](https://github.com/erlang/otp/wiki/Writing-good-commit-messages).
```

Comment:
I think part of the goal was that we'd inline these guidelines somehow and adjust them for our needs. In particular we don't really want to enforce 50 as a limit for the title.

Reply:
@annevk a bit confused. I'm attempting to follow the initial requests here. What needs changing and i'll do it ASAP. Don't want to take up too much of your time with this one. 

Please advise.
![capture d ecran 2017-09-25 a 10 22 02](https://user-images.githubusercontent.com/38223/30821832-879dbf82-a1db-11e7-873a-3b48b916d4c9.png)

Reply:
What @domenic says in your screenshot is to include the relevant portions from the Erlang wiki entry. That's what I was trying to repeat. To do that rather than link it. (Sorry for the delayed response, had a meeting all day yesterday.)